extensions/brew/package.json

```diff
@@ -56,6 +57,23 @@
       "required": false,
       "title": "Custom Brew Executable Path",
       "description": "Set this if your brew executable is in a non-standard location"
+    },
+    {
+      "name": "terminalApp",
+      "type": "dropdown",
+      "required": false,
+      "title": "Choose Terminal Application",
+      "data": [
+        {
+          "title": "Terminal",
+          "value": "terminal"
+        },
+        {
+          "title": "iTerm",
+          "value": "iterm"
+        }
+      ],
+      "description": "Choose the terminal application/emulator to use when using any Run in Terminal action."
```

Comment:
Also here: "The terminal application..."

Reply:
```suggestion
      "description": "The terminal application/emulator to use when using any Run in Terminal action."
```
👍